src/ReactiveUI.Validation.Tests/ReactiveUI.Validation.Tests.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="MSBuild.Sdk.Extras">
 
   <PropertyGroup>
-    <TargetFrameworks>netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks>netcoreapp3.1;net5</TargetFrameworks>
```

Comment:
net5.0 is the official framework name. It's the equivalent of netstandard20

Reply:
Hmm, they apparently have a lot of aliases for .NET 5 e.g. `net5.0`, `net5`, `netcoreapp5.0`.
Switched to use the official one `net5.0`.